nmtwizard/preprocess/tokenizer.py

```diff
@@ -11,6 +11,7 @@ def build_tokenizer(args):
     """Builds a tokenizer based on user arguments."""
     args = args.copy()
     args.pop('vocabulary', None)
+    args.pop('replace_vocab', None)
```

Comment:
Is this one needed even though `_get_vocab_info` is removing it?

Reply:
To my understanding, _get_vocab_info is activated after _build_data which uses the original tokenization field.

In new configurations 'replace_vocab' option is in a separate vocabulary field, so no problem.
In old configurations, however, it is supposed to be in 'tokenization' field, so I get an error in 'test_replace_vocab_old' test if I remove it.